.github/workflows/release.yml

```diff
@@ -4,15 +4,22 @@ on:
   push:
     tags:
       - "v*.*.*"
+    branches:
+      - main
 
 permissions:
   contents: read
 
+# Override the version if there is no tag release.
+env:
+  ELASTIC_CI_LOCAL_VERSION: ${{ startsWith(github.ref, 'refs/tags') && '' || github.run_id }}
```

Comment:
Do you need *ordering* of these published releases? Or is just any unique-ish identifier sufficient?

Reply:
I don't think we care about ordering or the packages built. My goal here is just to check that the github workflow for releases works fine.

Reply:
Oh, I guess this (`run_id`) works for ordering. All good.